AnkiConnect.py

```diff
@@ -502,11 +509,59 @@ def modelFieldNames(self, modelName):
                 return [field['name'] for field in model['flds']]
 
 
-    def multi(self, actions):
-        response = []
-        for item in actions:
-            response.append(AnkiConnect.handler(ac, item))
-        return response
+    def confForDeck(self, deck):
+        if not deck in self.deckNames():
+            return False
+
+        id = self.collection().decks.id(deck)
+        return self.collection().decks.confForDid(id)
+
+
+    def saveConf(self, conf):
```

Comment:
Use `getDeckConfig` to get the object, then modify it and save it again. This is an example in the project I'm writing:

![image](https://user-images.githubusercontent.com/4248177/29474536-045380d4-8454-11e7-81ad-5568a9f28bc0.png)

This changes the config of the default deck so that the intervals for new cards are as you can see, and the max number of new cards is allowed (the GUI options window doesn't allow higher than 9999). When saving, no more parameters are needed because the config ID is included with the object.